### DISCUSSION TITLE
Is there a docker Image for PyPGx

### DISCUSSION THREAD

**Discussion by mbc32** [pypgx/pypgx#85] [Q&A] [answered]
I can see a docker image in biocontainers (quay.io/biocontainers/pypgx) is this an official image for sbslee/pypgx?

Thanks,
Mikkel Christensen

**Comment by sbslee** [comment, marked as answer]
@mbc32,

Thanks for the question! That is NOT an official image for PyPGx. I have no idea who the uploader is, but I'd recommend that you install PyPGx via `conda` or `pip`. Please let me know if you have further questions.

Steven
